lib/dependencies/ImportParserPlugin.js

```diff
@@ -45,6 +47,20 @@ class ImportParserPlugin {
 					else
 						mode = importOptions.webpackMode;
 				}
+				if(typeof importOptions.webpackFilter !== "undefined") {
+					if(typeof importOptions.webpackFilter !== "string") {
```

Comment:
It would be super awesome, but not required, to add `RegExp` support to the parsing (see json5 module) to write `webpackExclude: /xyz/` instead of `webpackExclude: "xyz"`.